api/internal/service/user/repository/user_postgres.go

```diff
@@ -56,7 +56,20 @@ func (u *UserRepository) Register(ctx context.Context, input model.InputCreateUs
 }
 
 // Login user signin with return profile user
-func (u *UserRepository) Login(ctx context.Context, input model.InputLoginUser) (*ent.Profile, *ent.Role, error) {
+func (u *UserRepository) Login(ctx context.Context, input model.InputLoginUser) (*ent.User, *ent.Profile, *ent.Role, error) {
+	us, err := u.DB.User.
+		Query().
+		Where(
+			userAgregate.And(
+				userAgregate.EmailEQ(input.Email),
+				userAgregate.Password(input.Password),
+			),
+		).
+		Only(ctx)
+
+	if err != nil {
+		return nil, nil, nil, err
+	}
```

Comment:
i think we do not need a return detail user, because data user only email and password, you can make a session a username if the user success login